src/qt/walletframe.h

```diff
@@ -30,7 +30,7 @@ class WalletFrame : public QFrame
     void setClientModel(ClientModel *clientModel);
 
     bool addWallet(const QString& name, WalletModel *walletModel);
-    bool setCurrentWallet(const QString& name);
+    WalletModel* setCurrentWallet(const QString& name);
```

Comment:
Agree that this can be confusing. The setter will just return the referenced WalletModel pointer or `NULL` in case if the wallet was not found.

Added a comment.